plugins/inputs/logparser/logparser.go

```diff
@@ -131,9 +145,32 @@ func (l *LogParserPlugin) Start(acc telegraf.Accumulator) error {
 	defer l.Unlock()
 
 	l.acc = acc
-	l.lines = make(chan logEntry, 1000)
-	l.done = make(chan struct{})
-	l.tailers = make(map[string]*tail.Tail)
+	l.lines = make(chan *logline.LogLine)
```

Comment:
Really sorry about this, but actually let's just leave logparser on hpcloud/tail since I'm going to deprecate it anyway.